CONTRIBUTING.md

```diff
@@ -118,7 +118,7 @@ As part of the cleanup process, you should also run `make pretty-check` to ensur
 
 ```bash
 ./bootstrap
-./configure --enable-ftd --enable-cli --enable-diag --enable-dhcp6-client --enable-dhcp6-server --enable-commissioner --enable-joiner --with-examples=posix
+./configure --enable-cli-app=all --enable-diag --enable-dhcp6-client --enable-dhcp6-server --enable-commissioner --enable-joiner --with-examples=posix
```

Comment:
No options are actually required.  #1560 already includes an [update](https://github.com/openthread/openthread/pull/1560/files#diff-6a3371457528722a734f3c51d9238c13) to these instructions.